.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,44 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: ${{ github.event.comment.body == '/make build v1.20.14 ADM64' }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+
+      - name: Check outputs
+        run: |
+          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/k8sv1.20.14.tar.gz
+          tar -xvf k8sv1.20.14.tar.gz
+          wget https://dl.k8s.io/v1.20.14/kubernetes-server-linux-amd64.tar.gz
+          tar -xvf kubernetes-server-linux-amd64.tar.gz
+          sudo cp ./kubernetes/server/bin/kubectl ./rootfs/bin/
+          sudo cp ./kubernetes/server/bin/kubeadm ./rootfs/bin/
+          sudo cp ./kubernetes/server/bin/kubelet ./rootfs/bin/
+          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/sealer.tar.gz
+          sudo tar -xvf sealer.tar.gz -C /usr/bin
+          sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
+          cd ./rootfs
+          cat>imageList<<EOF
+          k8s.gcr.io/kube-apiserver:v1.20.14
```

Comment:
Using kubeadm image list to got the imageList

Reply:
```
kubeadm config images list --config kubeadm.yaml
```

need add --config  param to fixed kubernetes version .default fetch images version is latest version in 1.20.x

Reply:
The premise of using kubedm is to install kubedm. I think setting it to variable will make all versions render, and I have tested it